### PR TITLE
Add publication_ancillary assay type

### PR DESCRIPTION
**Pull request by jswelling**
This PR adds the publication_ancillary 'assay type' to assay_types.yaml with the goal of supporting visualization of publications.

### CODE REVIEW

**Comment by AlanSimmons** [comment]
Yes. I'll add it to the ontology.

**Comment by yuanzhou** [comment]
@AlanSimmons can you also approve this PR if it looks good to you? I normally wait to merge until all the reviewers approve.

**Review by AlanSimmons — APPROVED** [review]

**Review by yuanzhou — APPROVED** [review]

**Pull request merged**
